ansible/roles/logstash/templates/logstash-simple-2.conf

```diff
@@ -0,0 +1,150 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,e
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+input {
+    beats {
+        port => 5044
+    }
+
+    file {
+        path => "/media/ephemeral0/logs/accumulo/*.log"
```

Comment:
These paths should be referencing the `worker_data_dirs` variable instead of hard-coding.

Reply:
When I referenced the `worker_data-dirs` variable, this causes a pipeline error. `File` paths must be absolute, relative path specified:`I will be opening up a separate issue where this can be resolved.

Reply:
Created issue #370 for this

Reply:
In the interest of helping close things, I have addressed these in a [commit on my own fork](https://github.com/arvindshmicrosoft/fluo-muchos/commit/d2d6b2d20c36052796de5b908ca7bb2d477f9e4b) - could you please take a look at that, merge as appropriate and then push changes to this PR?